README.md

```diff
@@ -0,0 +1,19 @@
+# sbc-pkgbuilds 
+This repository contains Arch Linux PKGBUILDs to build kernel packages for SBCs with command `makepkgs`. The PKGBUILDs define how to build the packages, including dependencies, build steps, and installation steps.
+
+The build will grab the kernel source code from the [BredOS kernel repository]() and build the archlinux kernel package, which can be then incorporated into the BredOS image in the [BredOS images repository](https://github.com/BredOS/images).
```

Comment:
It's better off not being included at all, since we use lots of different sources for different boards.
```suggestion
```